tests/test-notify.cc

```diff
@@ -293,7 +293,7 @@ TEST_F(NotifyFixture, LevelsDuringBatteryDrain)
                                                      nullptr);
 
   // confirm that draining the battery puts
-  // the power_level change through its paces                                                              
```

Comment:
personally, I'd prefer functional changes and white-space fixup changes not being part of the same commit. It blurs what actually happens. White-space commits should be designated as such in the commit message IMHO.